.github/workflows/nvqc_regression_tests.yml

```diff
@@ -128,7 +128,8 @@ jobs:
             # nested_vectors: related to vector of pauli_words (https://github.com/NVIDIA/cuda-quantum/issues/1957)
             # custom_operation: https://github.com/NVIDIA/cuda-quantum/issues/1985
             # return_values: only supported in 0.8 NVQC service.
-            if [[ "$filename" != *"unsupport_args"* ]] && [[ "$filename" != *"state_overlap"* ]] && [[ "$filename" != *"compile_errors"* ]] && [[ "$filename" != *"nested_vectors"* ]] && [[ "$filename" != *"pauli_word"* ]] && [[ "$filename" != *"custom_operation"* ]] && [[ "$filename" != *"return_values"* ]]; then
+            # state_init: only supported in 0.8 NVQC service.
```

Comment:
Hopefully 0.8 will be deployed any day now. When it is deployed, will this change be needed?

Reply:
Not sure - it depends on how much of the new synthesis implementation is in 0.8 - it would need support for `-fkernel-exec-kind=2` flag, specifically this PR: https://github.com/NVIDIA/cuda-quantum/pull/2101 and before that.

Reply:
> Not sure - it depends on how much of the new synthesis implementation is in 0.8 - it would need support for `-fkernel-exec-kind=2` flag, specifically this PR: #2101 and before that.

Isn't synthesis running locally? If so, wouldn't a fully synthesized kernel be able to run on the NVQC server regardless of what version is on the server? (Perhaps it depends on the runtime functions used by that synthesized kernel?)

In any case, I'm mainly trying to figure out if we should create an issue to track the fact that we should remove this test from the exclusion list.

Reply:
Yes, the synthesis is running locally but the server will need new runtime API for creating states.
I added removal of it to https://github.com/NVIDIA/cuda-quantum/issues/1931.